### DISCUSSION TITLE
Setting media_type for ui.download

### DISCUSSION THREAD

**Discussion by me21** [zauberzeug/nicegui#2491] [Ideas / Feature Requests]
### Question

Hello,

Is it possible to specify the media type for `ui.download` command? For example, `'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'` to indicate the downloaded file is an Excel document.

I generate a Word document from the data entered by the user and would like to enable user to download that file. On another hand, the document can be easily recreated from the original data so I'm not sure I should save it to disk - maybe it's possible to keep it in memory?

**Comment by falkoschindler** [comment]
Hi @me21,

I think this isn't possible at the moment. But I just created PR https://github.com/zauberzeug/nicegui/pull/2494 to add this feature with the next release.

**Reply by falkoschindler** [reply]
This feature has been released with version 1.4.14.
